modules/op-installing-tkn-on-macos.adoc

```diff
@@ -10,7 +10,7 @@ For macOS, the `tkn` CLI is provided as a `tar.gz` archive.
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.17.2/tkn-macos-amd64-0.17.2.tar.gz[CLI].
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.19/tkn-macos-amd64-0.19.tar.gz[CLI].
```

Comment:
@VeereshAradhya  here too ?

Reply:
@ktania46 I would say yes in the previous releases they were the same through all platforms.